### PR TITLE
Week 7 Updates

### PR DESCRIPTION
**Pull request by NovaSagittarii**
- @advilm #14 
  - Fix jump behavior (coyote timer and double jump) @advilm 
- @advilm #13 
  - Implement dash ability, keybind: `SHIFT`.
- @NovaSagittarii #23 
  - Remove walking startup time. Fix block/dash softlock.
- @Dragonlinae #16 
  - Implement fast star attack and burst attack with indicators. Boss changes behavior when low on health.
- @Dragonlinae #18
  - Implement boss health bar, add intro sequence. 
- @Dragonlinae #19
  - Add boss NPC dialogue sequence. Interact keybind: `Q`.
- @Dragonlinae #21 
  - Add character profile in dialogue box.
- @AnPhu-D #20 
  - Implement combo system with timeout and damage bonus. (No GUI)
- @SafeDuck #22 
  - Implement moving platform with collision.

### CODE REVIEW

File: scripts/movement_state.gd [hunk 1]
```diff
@@ -32,15 +39,6 @@ func _init():
 	add_transition("Dash", "Moving")
 	add_transition("Dash", "Idle")
 
-func _ready():
-	coyote_timer = Timer.new()
```
Comment (advilm) [inline scripts/movement_state.gd:36]: If we are using the game tee method this shouldn't be here anymore

Comment (NovaSagittarii) [reply scripts/movement_state.gd:36]: its not one way or the other. this is simpler to merge to do since you did additional fixes :^)

Comment (NovaSagittarii) [reply scripts/movement_state.gd:36]: fixed in https://github.com/Dragonlinae/Papereaven/pull/17/commits/27d61af496f1b0a9877e87eb95ce430a64f4f3f1

**Review by AnPhu-D — APPROVED** [review]
Looks great. Thank you scrum master for your hard work!

**Pull request merged**